lesson_07/objects/objects_app/src/main/java/com/codedifferently/lesson7/jadatvapp/Tv.java

```diff
@@ -0,0 +1,81 @@
+package com.codedifferently.lesson7.jadatvapp;
+
+import java.util.List;
+
+public class Tv {
+  private String brand;
+  private String model;
+  private double screenSize;
+  private String resolution;
+  private DisplayTechnology displaytechnology;
+  private boolean isSmart;
+  private List<String> inputports;
+
+  public Tv(
+      String brand,
+      String model,
+      double screenSize,
+      String resolution,
+      DisplayTechnology displayTechnology,
+      boolean isSmart,
+      List<String> inputports) {
+    this.brand = brand;
+    this.model = model;
+    this.screenSize = screenSize;
+    this.resolution = resolution;
+    this.displaytechnology = displayTechnology;
+    this.isSmart = isSmart;
+  }
+
+  public String getBrand() {
+    return brand;
+  }
+
+  public void setBrand(String brand) {
+    this.brand = brand;
+  }
+
+  public String getModel() {
+    return model;
+  }
+
+  public void setModel(String model) {
+    this.model = model;
+  }
+
+  public double getScreenSize() {
+    return screenSize;
+  }
+
+  public void testSetScreenSize(double screenSize) throws TvException {
```

Comment:
This is exactly the same as `setScreenSize` below, please delete this one since the other method is named more accurately.

Reply:
Okay, I will try again but when I tried to delete it before making changes I kept getting an error.

Reply:
Maybe because you're also using it in a test and need to change it there as well.